### PR TITLE
Welcoming prompt to add my first item if my list is empty to help me get oriented to how the app works

### LINKED ISSUE #8: 7. As a user, I want to see a welcoming prompt to add my first item if my list is empty to help me get oriented to how the app works
AC:

* The list view, when there are no items to display, should show a prompt (e.g., a button) for the user to add their first item

### PR DESCRIPTION
**Pull request by AshlingH**
Issue: As a user, I want to see a welcoming prompt to add my first item if my list is empty to help me get oriented to how the app works 

AC: The list view, when there are no items to display, should show a prompt (e.g., a button) for the user to add their first item

Plan: 

- Set up welcome prompt- hover click input and hidden content detailing how the app works
-  Add CSS to make it look nice
-  Connect to add item.

### CODE REVIEW

**Review by jonbascos — COMMENTED** [review]
Good work you two!  I'm going to have to find out what htmlFor does.  Never seen it before.

**Comment by laurenmbeatty** [comment]
I wonder if this prompt is supposed to be on the Shopping List page...it seems like from the AC that if the user is on the Shopping List page, but doesn't have any items, it should show them a prompt to add their first item.

**Comment by jendevelops** [comment]
> I wonder if this prompt is supposed to be on the Shopping List page...it seems like from the AC that if the user is on the Shopping List page, but doesn't have any items, it should show them a prompt to add their first item.

Commits e5f7834 and a03ea0b address this issue. The button and message have been moved to the ShoppingList view.

File: src/pages/Home.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import React, { useState } from 'react';
-import '../App.css';
+/*import '../App.css';*/
```
Comment (dkimlim) [inline src/pages/Home.js:2]: could remove commented out App.css

**Review by dkimlim — COMMENTED** [review]
Good job, it's looking really nice :D

We might have some merge conflicts in ShoppingList this week, just a heads up!

File: src/pages/ShoppingList.js [hunk 2]
```diff
@@ -4,10 +4,40 @@ import fb from '../lib/firebase';
...
+				</div>
+			
+		);
+	};
+
 	//Thanks for sharing your code!!
 	useEffect(() => {
 		const db = fb.firestore();
-		const tokenRef = db.collection(token);
+		if(token){
```
Comment (laurenmbeatty) [inline src/pages/ShoppingList.js]: nice!

File: src/pages/ShoppingList.js [hunk 3]
```diff
@@ -4,10 +4,40 @@ import fb from '../lib/firebase';
...
+	const welcomeInstructions = () => {
+		return (
+			<div>
+				<input
+						type="checkbox"
+						className="button-link"
+						id="WelcomeClick"
+					/>
+					<label htmlFor="WelcomeClick" id="Welcome">
+					You're list looks empty. Need help?
```
Comment (laurenmbeatty) [inline src/pages/ShoppingList.js]: nice use of `htmlFor`

Comment (laurenmbeatty) [inline src/pages/ShoppingList.js]:
```suggestion
					Your list looks empty. Need help?
```

**Review by laurenmbeatty — CHANGES_REQUESTED** [review]
Just that one spelling fix, then you're good to go.

**Comment by jendevelops** [comment]
Closes #8

**Pull request merged**